### PR TITLE
Vatgrown/Vatborn sprites attempt [WIP]

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Adds custom sprites for vatborns. WIP.

## Why It's Good For The Game
Because they're snowflakes.

## Changelog
:cl:
add: Vatborns/Vatgrowns now have unique sprites.
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
I still have problems with this, especially relating to the hair of the vat people.

**Comment by XSlayer300** [comment]
I feel like this PR is already good as it is.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -298,8 +298,11 @@
 	/*abilities = list(/client/proc/test_ability)*/
 
 /datum/species/human/vatborn
-	name = "Vatborn"
-	name_plural = "Vatborns"
+	name = "Vat-Born Human"
+	name_plural = "Vat-Born Humans"
+	icobase = 'icons/mob/human_races/r_vatborn.dmi'
+	deform = 'icons/mob/human_races/r_vatborn.dmi'
+	species_flags = HAS_SKIN_TONE|HAS_LIPS|HAS_UNDERWEAR
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/species.dm:305]: I wouldnt be against making vatborn have the yellow human sprites and the vatgrown having the pale sprites or the other way around

Would help with visual differentiation

Comment (TiviPlus) [reply code/modules/mob/living/carbon/human/species.dm:305]: @XSlayer300 ?

**Pull request closed without merging**